src/parsley.js

```diff
@@ -3,112 +3,97 @@
 //     (c) 2012-2015 Guillaume Potier, Wisembly
 //     Parsley may be freely distributed under the MIT license.
 
-define([
-
-  // ### Requirements
-
-  // Handy third party functions
-  'parsley/utils',
-  // Parsley default configuration
-  'parsley/defaults',
-  // An abstract class shared by `ParsleyField` and `ParsleyForm`
-  'parsley/abstract',
-  // A registry of all Parsley validators (built-in and custom)
-  'parsley/validator_registry',
-  // `ParsleyUI` static class. Handles all UI and UX
-  'parsley/ui',
-  // `ParsleyForm` Class. Handles form validation
-  'parsley/form',
-  // `ParsleyField` Class. Handles field validation
-  'parsley/field',
-  // `Multiple` Class. Extend `ParsleyField` to generate `ParsleyFieldMultiple`
-  'parsley/multiple',
-  // Factory to create Parsley instances (Form, Field or FieldMultiple)
-  'parsley/factory',
-  // Tiny Parsley Pub / Sub mechanism, used for `ParsleyUI` and Listeners
-  'parsley/pubsub',
-  // Default en constraints messages
-  'i18n/en'
-], function (ParsleyUtils, ParsleyDefaults, ParsleyAbstract, ParsleyValidatorRegistry, ParsleyUI, ParsleyForm, ParsleyField, ParsleyMultiple, ParsleyFactory) {
-
-  // Inherit `on`, `off` & `trigger` to Parsley:
-  var Parsley = $.extend(new ParsleyAbstract(), {
-      $element: $(document),
-      actualizeOptions: null,
-      _resetOptions: null,
-      Factory: ParsleyFactory,
-      version: '@@version'
-    });
-
-  // Supplement ParsleyField and Form with ParsleyAbstract
-  // This way, the constructors will have access to those methods
-  $.extend(ParsleyField.prototype, ParsleyAbstract.prototype);
-  $.extend(ParsleyForm.prototype, ParsleyAbstract.prototype);
-  // Inherit actualizeOptions and _resetOptions:
-  $.extend(ParsleyFactory.prototype, ParsleyAbstract.prototype);
-
-  // ### jQuery API
-  // `$('.elem').parsley(options)` or `$('.elem').psly(options)`
-  $.fn.parsley = $.fn.psly = function (options) {
-    if (this.length > 1) {
-      var instances = [];
-
-      this.each(function () {
-        instances.push($(this).parsley(options));
-      });
-
-      return instances;
-    }
-
-    // Return undefined if applied to non existing DOM element
-    if (!$(this).length) {
-      ParsleyUtils.warn('You must bind Parsley on an existing element.');
+import $ from 'jQuery';
```

Comment:
Thx for the quick feedback. I made the tweaks you pointed out, still won't load properly though... Note that the "test" is just the boilerplate test requiring our lib...

```
[~/parsley(es6)]$ gulp
[DEPRECATION NOTICE] Esperanto is no longer under active development. To convert an ES6 module to another format, consider using Babel (https://babeljs.io)
[22:28:08] Using gulpfile ~/parsley/gulpfile.js
[22:28:08] Starting 'lint-src'...
[22:28:08] Starting 'lint-test'...
[22:28:08] Finished 'lint-test' after 82 ms
[22:28:08] Finished 'lint-src' after 385 ms
[22:28:08] Starting 'test'...
[22:28:10] 'test' errored after 1.45 s
[22:28:10] Error in plugin 'gulp-mocha'
Message:
    Cannot find module 'jquery'
Details:
    code: MODULE_NOT_FOUND
Stack:
Error: Cannot find module 'jquery'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/Users/mal/parsley/src/parsley.js:12:15)
    at Module._compile (module.js:460:26)
    at normalLoader (/Users/mal/parsley/node_modules/babel-core/lib/api/register/node.js:199:5)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/mal/parsley/node_modules/babel-core/lib/api/register/node.js:216:7)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
```

Any idea?